skydb/pq/builder.go

```diff
@@ -262,19 +262,17 @@ func (p accessPredicateSqlizer) ToSql() (sql string, args []interface{}, err err
 		if err != nil {
 			panic("unexpected serialize error on role")
 		}
-		b.WriteString(`_access @> '[{"role":`)
-		b.Write(escapedRole)
-		b.WriteString(`}]' OR `)
+		b.WriteString(fmt.Sprintf(`_access @> '[{"role": %s}]' OR `, escapedRole))
 	}
-	b.WriteString(`_access @> '[{"user_id":`)
-	escapedID, _ := json.Marshal(p.user.ID)
-	if err != nil {
-		panic("unexpected serialize error on user_id")
-	}
-	b.Write(escapedID)
-	b.WriteString(`}]' OR _access IS NULL OR _owner_id = ?)`)
+	b.WriteString(fmt.Sprintf(`_access @> '[{"user_id": "%s"}]' OR `, p.user.ID))
```

Comment:
`json.Marshal` is used to escaped the `user_id`

If you use fmt.Sprintf and user id is string like `oursky"rickmak"`, the SQL will not able to execute.